frank-doc-frontend/src/environments/environment.ts

```diff
@@ -3,6 +3,7 @@ export const environment = {
   javadocBaseUrl: 'https://javadoc.frankframework.org',
   githubWikiBaseUrl: 'https://github.com/frankframework/frankframework/wiki',
   frankDocUrl: 'js/frankdoc.json',
+  applicationName: 'FF! Doc',
```

Comment:
Maybe also use this name in the header? (but then without `FF! `)

Reply:
Do you mean next to the logo?